app/src/main/java/com/tari/android/wallet/ui/extension/ViewExtensions.kt

```diff
@@ -247,7 +220,117 @@ internal fun View.drawable(@DrawableRes id: Int): Drawable? = context.drawable(i
 internal class ThrottleClick(private val delegate: (View) -> Unit) :
     View.OnClickListener {
     override fun onClick(v: View?) {
-        v?.let(UIUtil::temporarilyDisableClick)
+        v?.temporarilyDisableClick()
         v?.let(delegate)
     }
 }
+
+// method name is this rather than "setWidth" because "setWidth" might conflict with View's
+// subtypes intrinsic methods
+fun View.updateWidth(width: Int) {
```

Comment:
Let's rename it to `setWidth` so it's consisten with the rest (`setBottomMargin` etc).